problems/removeVowels.js

```diff
@@ -6,8 +6,9 @@
 * @returns {string} - Returns a new string without any vowels.
 */
 
-function removeVowels() {
-
+function removeVowels(str) {
+    return str.replace(/[aeiou]/gi, "")
```

Comment:
This answer is very very slick, but make sure you understand 100% how this RegEx is working and could replicate this answer on an interview.